src/services/accounts/AccountsValidateService.spec.ts

```diff
@@ -51,6 +60,9 @@ describe('Validate addresses', () => {
 		const expectedResponse = {
 			isValid: true,
 			ss58Prefix: '16',
+			network: 'kulupu',
```

Comment:
Just wondering how many third party chains we support here; any which has claimed an ss58 prefix?

Reply:
Yup exactly, all the chains that have an ss58 prefix, but I also believe it's constrained to what `allNetworks` has logged in polkadot-js.